.github/workflows/build.yml

```diff
@@ -888,7 +888,79 @@ jobs:
           bash tools/export_mongodb.sh
           tar -cvf - *.json | aws s3 cp - s3://${{ secrets.AWS_LOG_BUCKET_NAME }}/core-pipeline/${{ github.run_number }}/${{ github.run_attempt }}/connectivity-ca${{ matrix.ca }}-database.tar.gz
 
+  arm-instances:
+    needs:
+      - versionning
+      - images
+    env:
+      VERSION: ${{ needs.versionning.outputs.version }}
+    runs-on: ubuntu-24.04-arm
+    steps:
+      - name: Checkout
+        uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Install AWSCLI (the one in the Github runner does not work)
+        run: |
+          pip install awscli
+
+      - name: Setup just
+        uses: taiki-e/install-action@14083e64ac8cf1f5e54356df00b9779b23e192a1 # v2
+        with:
+          tool: just
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@b9cd54a3c349d3f38e8881555d616ced269862dd # v3
+        with:
+          terraform_version: latest
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@184bdaa0721073962dff0199f1fb9940f07167d1 # v3
+        with:
+          username: ${{ secrets.DOCKER_HUB_LOGIN }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+
+      - name: Deploy Core
+        run: |
+          MONITOR_PREFIX="monitor/deploy/" tools/retry.sh -w 30 -- tools/monitor.sh \
+            just log_level=Information tag=${VERSION} queue=activemq worker=bench deploy
+          sleep 10
 
+      - name: Run Bench test tasks - many tasks (400)
+        timeout-minutes: 3
+        run: |
+          MONITOR_PREFIX="monitor/bench-many-tasks-400/" tools/monitor.sh \
+            docker run --net armonik_network --rm \
+            -e BenchOptions__NTasks=400 \
+            -e BenchOptions__TaskDurationMs=10 \
+            -e GrpcClient__Endpoint=http://armonik.control.submitter:1080 \
+            dockerhubaneo/armonik_core_bench_test_client:$VERSION
+      - name: Show logs
+        if: always()
+        run: |
+          export AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }}
+          export AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          docker cp fluentd:/armonik-logs.json - | gzip -c | aws s3 cp - s3://${{ secrets.AWS_LOG_BUCKET_NAME }}/core-pipeline/${{ github.run_number }}/${{ github.run_attempt }}/${{ github.job }}.json.gz
```

Comment:
can you add to the file name something like "arm" to easily  identify them in the bucket

Reply:
that's already there `${{ github.job }}` should be `arm-instances`